poodle/src/components/Mypage/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { FC } from 'react';
+import * as S from '@/styles/Mypage/ProgressBar'
+
+const dummyData = {
+    '전형구분': 'false',
```

Comment:
```suggestion
    '전형구분': 'false',
```
value 가 boolean 이 아니라 string 으로 넘어오나요?

Reply:
api docs가 안나왔지만 99프로 boolean으로 올것 같네요.
![image](https://user-images.githubusercontent.com/48552260/81491175-df40cd00-92c5-11ea-801f-6cb2fabc18ff.png)
boolean형태로 styled components에 prop로 넘겨주니 사진과 같은 에러가 떠서 이를 해결하고자 string으로 코드를 짠것 같습니다.
이에 대해 구글링을 더 해보니 styled components는 props를 dom에 그대로 넘겨주고(?) dom 엘리먼트는 boolean속성을 가질수 없다해서 이를 어떻게 해결할지 고민하다가 
`<S.ProgressImageLink to={{ pathname: "전형구분", state: { isfinished: isfinished } }} />`라는 방식으로 수정해보았는데 더 나은 방법이 있나요? 잘 모르겠습니다...